src/vtquery.cpp

```diff
@@ -324,12 +342,12 @@ struct Worker : Nan::AsyncWorker {
 
             // create properties object
             v8::Local<v8::Object> properties_obj = Nan::New<v8::Object>();
-            v8::Local<v8::Object> tilequery_properties_obj = Nan::New<v8::Object>();
             for (auto const& prop : feature->properties) {
-                properties_obj->Set(Nan::New<v8::String>(prop.first).ToLocalChecked(), get_property_value(prop.second));
+                set_property(prop, properties_obj);
             }
 
             // set properties.tilquery
+            v8::Local<v8::Object> tilequery_properties_obj = Nan::New<v8::Object>();
             tilequery_properties_obj->Set(Nan::New("distance").ToLocalChecked(), Nan::New<v8::Number>(feature->distance));
             std::string og_geom = getGeomTypeString(feature->original_geometry_type);
             tilequery_properties_obj->Set(Nan::New("geometry").ToLocalChecked(), Nan::New<v8::String>(og_geom).ToLocalChecked());
```

Comment:
@flippmoke The original geometry type string shows up in the final GeoJSON, but `layer_name` doesn't (one line lower). Not sure why.

Reply:
This has been fixed